Docs work (#194)
MystenLabs/suins-contracts: documentation/pages/move-registry.mdx

# Move Registry

> Move Registry is built to provide a name service for interacting / building with packages from the ecosystem. You could consider it the name service for packages.

Reviewer: ```suggestion
Move Registry provides a name service for interacting and building with packages from the ecosystem.
```

Move Registry is currently used for the following scenarios: 

1. To reference packages and types in PTBs using their names.
2. To depend on other packages when developing with Move.

> ## Why would I use the Move Registry?

Reviewer: ```suggestion
## Why use the Move Registry?
```

> Move Registry provides a uniform (single source of truth) naming service for packages in the Move ecosystem. 

Reviewer: Probably this should be in introduction as well.

> This means that you can reference packages by their names, and the Move Registry will resolve the package address for you, despite the network*.

Reviewer: ```suggestion
It allows to reference packages by their names, and the Move Registry will resolve the package address, despite the network.
```

Reviewer 2: While we're here could we change "despite" to "regardless of"?

Reviewer 3: How's about "irregardless"...

> Additionally, Move Registry helps with working with versioning of packages. You can call a specific version of a package easily, without having to 
> resolve the addresses yourself, or worry about the package being updated (name without a specified version defaults to the latest).

Reviewer: ```suggestion
Additionally, Move Registry helps with working with versioning of packages. Call a specific version of a package easily, without having to 
resolve the addresses or worry about the package being updated. For example, name without a specified version defaults to the latest.
```

> *Move Registry default service currently supports Mainnet and testnet.*

Reviewer: ```suggestion
*Move Registry default service currently supports mainnet and testnet neworks.*
```
